docs/api/covidcast-signals/doctor-visits.md

```diff
@@ -33,15 +33,20 @@ percentage of COVID-related doctor's visits in a given location, on a given day.
 Note that because doctor's visits may be reported to the health system partners
 several days after they occur, these signals are typically available with
 several days of lag. This means that estimates for a specific day are only
-available several days later.
+available several days later. 
 
 The amount of lag in reporting can vary, and not all visits are reported with
 the same lag. After we first report estimates for a specific date, further data
 may arrive about outpatients visits on that date. When this occurs, we issue new
-estimates for those dates. This means that a reported estimate for, say, June
+estimates for those dates to backfill any missing data. This means that a reported estimate for, say, June
```

Comment:
We usually use "backfill" as a noun to describe the phenomenon of data revisioning. We could extend the definition of "backfill" to be a verb as well, or say something like "we issue new estimates for those dates that include the most recent data reports."